lib/class-wp-theme-json.php

```diff
@@ -92,6 +92,7 @@ class WP_Theme_JSON {
 			'lineHeight'     => null,
 			'textDecoration' => null,
 			'textTransform'  => null,
+			'letterSpacing'  => null,
```

Comment:
Small nitpick -- it looks like this list is alphabetized. Should we maybe put `letterSpacing` before `lineHeight`?